logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/conversation/ClearConversationContentTest.kt

```diff
@@ -136,17 +137,17 @@ class ClearConversationContentTest {
             verify(assetRepository)
                 .suspendFunction(assetRepository::deleteAsset)
                 .with(anything(), anything(), anything())
-                .wasInvoked()
+                .wasNotInvoked()
```

Comment:
same here

Reply:
same, as above this test is if message delete failed so no assets are deleted